swlor2_2da/portraits.2da

```diff
@@ -4432,3 +4432,4 @@
 4428   droF8_         1      167    ****            0      ****          
 4429   droF9_         1      167    ****            0      ****          
 4430   droF10_        1      167    ****            0      ****          
+4431   Kat_           1      167    ****            0      ****  
```

Comment:
2DA has this listed as Kat_ but the files are po_KatL

The po_ is prefixed as it should be, but there's a missing underscore in the file names. I don't think this will show up in-game, can you confirm with a screenshot?